sphinx-jsonschema/__init__.py

```diff
@@ -228,9 +228,9 @@ def get_json_data(self):
         elif filename and filename.startswith('http'):
             # Appears to be URL so process it as such
             schema, source = self.from_url(filename)
-        elif os.path.exists(filename):
+        elif os.path.exists(source := self._convert_filename(filename)):
```

Comment:
This requires Python 3.8, so matching `python_requires` should be added to `setup.py` to reflect his.

Reply:
f94a5132d76cdea3cf3a0447fa0e1a8ddd486964

Reply:
Will undo, see @kimberninger's comments below